lib/file_items_gateway.rb

```diff
@@ -41,4 +41,10 @@ def save(items)
   def delete_all
     File.unlink(FILE_PATH) if File.exist?(FILE_PATH)
   end
+
+  def delete_row(index)
+    items = all
+    items.delete(items[index])
```

Comment:
would items.delete_at(index) also work here?

Reply:
delete is a built-in method to delete elements from array